app/src/main/java/com/practicum/playlistmaker/App.kt

```diff
@@ -0,0 +1,31 @@
+
+
+package com.practicum.playlistmaker
+
+import android.app.Application
+import androidx.appcompat.app.AppCompatDelegate
+
+class App : Application() {
+
+    var darkTheme = false
```

Comment:
Лучше эту переменную сделать приватной, либо сделать ей приватный сеттер:
`
```suggestion
    var darkTheme = false
        private set
```
Чтобы нельзя было сломать логику работы с темой снаружи этого класса